pyscriptjs/tests/integration/test_plugins.py

```diff
@@ -186,3 +186,115 @@ def test_no_plugin_attribute_error(self):
         )
         # EXPECT an error for the missing attribute
         assert error_msg in self.console.error.lines
+
+    def test_fetch_python_plugin(self):
+        """
+        Test that we can fetch a plugin from a remote URL. Note we need to use
+        the 'raw' URL for the plugin, otherwise the request will be rejected
+        by cors policy.
+        """
+        self.pyscript_run(
+            """
+            <py-config>
+                plugins = [
+                    "https://raw.githubusercontent.com/FabioRosado/pyscript-plugins/main/python/hello-world.py"
```

Comment:
Is there a way that this content could be source from the a repo under the PyScript org, rather than your personal 😁?

Reply:
Yeah I think it would be good to put these in the plugins repo. I'll open a PR with them and then I can change the URLs 😄

Reply:
I need to do it in a follow up because I don't have permission to push to the plugins repo 😄

Reply:
Makes sense to me!